src/ifElse.test.js

```diff
@@ -1,11 +1,49 @@
 'use strict';
 
+const { ifElse } = require('./ifElse');
+
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  it('should run the first callback if the condition returns true', () => {
+    const condition = jest.fn(() => true);
+    const first = jest.fn();
+    const second = jest.fn();
+
+    ifElse(condition, first, second);
+
+    expect(condition).toHaveBeenCalled();
+    expect(first).toHaveBeenCalled();
+    expect(second).not.toHaveBeenCalled();
+  });
+
+  it('should run the second callback if the condition returns false', () => {
+    const condition = jest.fn(() => false);
+    const first = jest.fn();
+    const second = jest.fn();
+
+    ifElse(condition, first, second);
 
-  it('should ', () => {
+    expect(condition).toHaveBeenCalled();
+    expect(first).not.toHaveBeenCalled();
+    expect(second).toHaveBeenCalled();
+  });
+
+  it('should not return any result', () => {
```

Comment:
describe what are you tested
should be understandable why it should not return result